gcp/functions/pypi/main.py

```diff
@@ -59,15 +71,21 @@ def publish(event, context):
         elif evt.fixed:
           events.append({'fixed': evt.fixed})
 
-    request = json.dumps([{
-        'id': vulnerability.id,
-        'project': affected.package.name,
-        'versions': list(affected.versions),
-        'link': f'https://osv.dev/vulnerability/{vulnerability.id}',
-        'aliases': list(vulnerability.aliases),
-        'details': vulnerability.details,
-        'events': events,
-    }]).encode()
+    versions.extend(affected.versions)
+
+  if vulnerability.HasField('withdrawn'):
+    events = []
```

Comment:
@di this is the expected behaviour right?

Reply:
We don't really care about `events` unless the vulnerability is marked as fixed, the important thing is `versions = []` on the line below 👍